flask_reporting.py

```diff
@@ -0,0 +1,94 @@
+import os
```

Comment:
so now we have some bash, R, sql and python for one script? isn't overkilled or showing that maybe something else should be done from scratch rather than patching?

Reply:
Completely optional as described in the readme . You can still run the bash script.